src/Scaffold_Command.php

```diff
@@ -902,16 +902,19 @@ private function pluralize( $word ) {
 			'/$/'                      => 's'
 		);
 
-		$uncountable = array( 'equipment', 'information', 'rice', 'money', 'species', 'series', 'fish', 'sheep' );
-
+		$uncountable = array('sheep', 'fish','deer','series','species','money','rice','information','equipment');
```

Comment:
@Umangvaghela While you're working on this, can you please update this as per WordPress' coding standards?

1. Space after `,`
2. Space after `(` and before `)`

More details about coding standards can be found here - 
https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/